skia-bindings/build_support/skia/config.rs

```diff
@@ -27,19 +28,34 @@ pub struct BuildConfiguration {
 
     /// C++ compiler to use
     cxx: String,
+
+    /// Optional target
+    pub target: Option<Target>,
 }
 
 /// Builds a Skia configuration from a Features set.
 impl BuildConfiguration {
     pub fn from_features(features: features::Features, skia_debug: bool) -> Self {
+        let cc = cargo::env_var("CLANGCC")
+            .or_else(|| cargo::env_var("CC"))
+            .unwrap_or_else(|| "clang".to_string());
+        let target = cc.find("--target=").and_then(|target_option_offset| {
+            cc[(target_option_offset + "--target=".len())..]
+                .split_once(' ')
+                .map(|(target_str, ..)| crate::build_support::cargo::parse_target(target_str))
```

Comment:
`cargo::parse_target` should be enough to resolve the function.

Reply:
Hmm, I'm not sure that's possible because `split_once()` returns a tuple and we're interested in only the first field of the tuple. Or is there a way to resolve against just the function even though it doesn't accept a tuple?

Reply:
I meant to just remove the `crate::build_support::` qualification prefix, because the module `cargo` is already in scope.

Reply:
Got it, thanks :)